scripts/set_release_number.sh

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

Reply:
Have you looked for mix tasks that do this? There seem to be a few things up in hex for this kind of thing.

Reply:
No but I just copied the CI stuff from off_broadway_kafka because it was 3 am and I wanted the newest version of brook to unblock umbrella migration. :shrug:

Reply:
Do you feel strongly about this shebang change? I generally avoid that on remote systems to avoid someone aliasing "bash" to "do some funky stuff that exports all your credit card data" unless i explicitly control the system.

Reply:
No I don't feel strongly.